web/src/popover_menus.js

```diff
@@ -81,6 +94,16 @@ export function starred_messages_sidebar_menu_handle_keyboard(key) {
     popovers.popover_items_handle_keyboard(key, items);
 }
 
+export function all_messages_sidebar_menu_handle_keyboard(key) {
```

Comment:
Should we consider refactoring the code, including the hotkey.js file? It seems that keeping it DRY would be beneficial.